releasenotes/acs-emailapi-releasenote.md

```diff
@@ -0,0 +1,25 @@
+# Release notes for Email API
+
+## Email 
+Azure Communication Services new Email APIs to enable application-to-person (A2P) communication. 
+[new SDKs available]( https://docs.microsoft.com/azure/communication-services/concepts/sdk-features) for the following Languages:
```

Comment:
>https://docs.microsoft.com/azure/communication-services/concepts/sdk-features [](http://example.com/codeflow?start=22&length=77)

just FYI i got a 404 when i click on this link. Maybe just permissions issue on my end?